ansible/library/load_extra_dpu_config.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/python
+
+import paramiko
+import os
+import time
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.smartswitch_utils import smartswitch_hwsku_config
+
+DPU_HOST_IP_BASE = "169.254.200.{}"
```

Comment:
>169.254.200 [](http://example.com/codeflow?start=20&length=11)

are you able to get the DPU_HOST_IP_BASE from a ground truth? Is it from buildimage repo?

Reply:
No. It is static and agreed in the design.